simple_app/templates/base.html

```diff
@@ -23,8 +23,8 @@
     <script type="text/javascript" src="{{ url_for('static', filename='js/bootstrap.js') }}"></script>
 
     <!--Bokeh -->
-    <script type="text/javascript" src="{{ url_for('static', filename='js/bokeh-2.3.2.min.js') }}"></script>
-    <script type="text/javascript" src="{{ url_for('static', filename='js/bokeh-widgets-2.3.2.min.js') }}"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/bokeh-2.4.0.min.js') }}"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/bokeh-widgets-2.4.0.min.js') }}"></script>
```

Comment:
I've seen other examples that use things like:
```
<script src="https://cdn.bokeh.org/bokeh/release/bokeh-2.2.3.min.js"
        crossorigin="anonymous"></script>
<script src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.2.3.min.js"
        crossorigin="anonymous"></script>
<script src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.2.3.min.js"
        crossorigin="anonymous"></script>
```
to avoid having to have the bokeh JS files in their applications. Any reason why we shouldn't do something similar?

Reply:
PyCharm told me to.. It does also improve load times and ensures those resources are always accessible & static (e.g. if their host server is down or intercepted -- which wouldn't be obvious to an end user, that wouldn't affect us). I believe it's generally good practice to have local versions of things